sample.config.yaml

```diff
@@ -2,16 +2,32 @@ bridge:
   # Port to host the bridge on
   # Used for communication between the homeserver and the bridge
   port: 8434
+
   # The host connections to the bridge's webserver are allowed from
   bindAddress: localhost
+
   # Public domain of the homeserver
   domain: matrix.org
+
   # Reachable URL of the Matrix homeserver
   homeserverUrl: https://matrix.org
 
+  # Display name of the bridge bot
+  displayname: Discord Puppet Bridge
+
+  # Avatar URL of the bridge bot
+  avatarUrl: mxc://example.com/1ef7ef86a5b2b283f01b7236b54ec8af
```

Comment:
maybe not have a non-existing mxc uri as default without commenting it out? :)

Reply:
Ahh, yes.